docs/conf/portland/2021/writing-day.rst

```diff
@@ -24,8 +24,30 @@ The `Good Docs Project <https://thegooddocsproject.dev/>`_ is an open source com
 - **Easy button for docs setup** - One of the biggest hurdles to creating good documentation is setting up your initial documentation toolchain. Come collaborate with our docs tool experts who are working on creating a Hugo/Docsy repository that can be easily cloned to set up a new documentation system for a project.
 - **Industrial fiction** - Our goal is to provide good examples of our project's templates being used in an actual project. To accomplish this goal, we're creating our own fake open source project that can be used to generate documentation examples. Come help us build both the world and the documentation for The Chronologue, a fake open source tool that helps future time travelers sync their clocks across multiple dimensions.
 
+Trino project
+^^^^^^^^^^^^^
+
+`Trino <https://trino.io>`_ is a fast distributed SQL query engine for big data
+analytics. We’re excited to join you Write the Docs, this is our first `Writing Day <https://trino.io/blog/2021/04/14/wtd-writing-day.html>`_! We’ll have Trino
+enthusiasts available to help you get started, we even made `a video to help you get started <https://youtu.be/yseFM3ZI2ro>`_.
+
+Here’s a few ways you can contribute to the Trino open source project:
+
+- `Review our doc readme <https://github.com/trinodb/trino/tree/master/docs>`_, 
+   perfect for first time contributors.
+- `Improve our docs <https://trino.io/docs/current/>`_, we made an `issues filter <https://github.com/trinodb/trino/issues?q=is%3Aopen+label%3Adocs+label%3A%22good+first+issue%22>`_ to help you find ways to contribute. 
+   - Edit an existing document, perfect for
```

Comment:
Why the short lines here in the bullet list?

Reply:
It's supposed to be the bullets under the "Improve our docs" section.